README.md

```diff
@@ -73,6 +73,26 @@ Suppose we want to add a Kubic image. A suitable Vagrant box for such an image i
 
 By the way, more info on how to use Kubic images with vagrant-ceph is available in the [openSUSE:Ceph Wiki](https://en.opensuse.org/openSUSE:Ceph#Using_Rook_in_Vagrant_cluster).
 
+Following are two more examples for adding Vagrant boxes for openSUSE Leap 15.1 and openSUSE Tumbleweed, from the openSUSE download server:
+
+* __openSUSE Leap 15.1__
+
+`vagrant box add --provider libvirt --name virt-appl/openSUSE-Leap-15.1 https://download.opensuse.org/repositories/Virtualization:/Appliances:/Images:/openSUSE-Leap-15.1/images/Leap-15.1.x86_64-libvirt.box`
+
+* __openSUSE Tumbleweed__
+
+`vagrant box add --provider libvirt --name virt-appl/Tumbleweed https://download.opensuse.org/repositories/Virtualization:/Appliances:/Images:/openSUSE-Tumbleweed/openSUSE_Tumbleweed/Tumbleweed.x86_64-libvirt.box` 
```

Comment:
I don't think these are the "official" boxes for Tumbleweed. The official boxes are available on vagrantcloud (and are links to OBS. eg the current box is http://ftp.tu-chemnitz.de/pub/linux/opensuse/tumbleweed/appliances/Tumbleweed.x86_64-1.0-libvirt-Snapshot20191018.vagrant.libvirt.box). See https://www.suse.com/c/vagrant-boxes-with-opensuse-tumbleweed-check-it-out/ .
Btw. I already did the switch for Tumbleweed in https://github.com/openSUSE/vagrant-ceph/pull/56 . I was not aware that you will work on that.

Reply:
Hm, yes, "official" may be a poor choice of a word. ATM, in VagrantCloud there are only Tumbleweed images.

Reply:
`[Research] [Help wanted] SLES-15 box now available too (was: Please test the new openSUSE Vagrant boxes)`
there was a thread in research, that actually describe how boxes from Virtualization got to vagrantcloud.

so Tumbleweed should be 1to1 in the cloud and on OBS.

Reply:
> so Tumbleweed should be 1to1 in the cloud and on OBS.

yes. it's just a redirect to OBS. But to the official project (under the openSUSE:Factory project) and not to the devel project (which is Virtualization:Appliances:Images:openSUSE-Tumbleweed)